src/controllers/controlpickermenu.cpp

```diff
@@ -767,20 +826,55 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     addControl("[EffectRack1]", "show",
                tr("Effect Rack Show/Hide"),
                tr("Show/hide the effect rack"), guiMenu);
-    addControl("[Library]", "show_coverart",
-               tr("Cover Art Show/Hide"),
-               tr("Show/hide cover art"), guiMenu);
+    addControl("[Skin]",
+            "show_4effectunits",
+            tr("4 Effect Units Show/Hide"),
+            tr("Switches between showing 2 and 4 effect units"),
+            guiMenu);
```

Comment:
Do these exist for all skins?

Reply:
all except Shade ;)
We have one case IINM where it's the other way around: show_spinny1 or something that only exists in Shade, in my recent MIDI PR.
I added a comment to place a hint about this in the Wizard dialog.